projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -111,53 +380,70 @@ export function checkGlobalMessageDisplayed(isDisplayed: boolean): void {
 }
 
 /**
- * Checks submit button on quote page.
+ * Checks "Submit" button o n the quote details overview page
  */
-export function checkSubmitButton(isEnabled: boolean): void {
+export function checkSubmitBtn(isEnabled: boolean): void {
+  log(
+    'Checks "Submit" button o n the quote details overview page',
+    checkSubmitBtn.name
+  );
   if (isEnabled) {
-    cy.get('button.btn-primary').should('be.enabled');
+    cy.get('button.btn-primary').contains(SUBMIT_BTN).should('be.enabled');
   } else {
-    cy.get('button.btn-primary').should('be.disabled');
+    cy.get('button.btn-primary').contains(SUBMIT_BTN).should('be.disabled');
   }
 }
 
+/**
+ * Checks that comments are no longer editable and the input field doesnt exist anymore.
+ */
 export function checkCommentsNotEditable(): void {
-  cy.get('cx-quote-details-comment .cx-message-input').should('not.exist');
+  trace(checkCommentsNotEditable.name, () =>
+    cy.get('cx-quote-details-comment .cx-message-input').should('not.exist')
+  );
 }
 
 /**
  * Checks presence of quote list
  */
 export function checkQuoteListPresent() {
-  cy.get('cx-quote-list').should('exist');
+  trace(checkQuoteListPresent.name, () =>
+    cy.get('cx-quote-list').should('exist')
+  );
 }
 
 /**
  * Navigates to quote list via my account
  */
 export function navigateToQuoteListFromMyAccount() {
-  cy.get('cx-page-layout[section="header"]').within(() => {
-    cy.get('cx-navigation-ui.accNavComponent')
-      .should('contain.text', 'My Account')
-      .and('be.visible')
-      .within(() => {
-        cy.get('nav > ul > li > button').first().focus().trigger('keydown', {
-          key: ' ',
-          code: 'Space',
-          force: true,
+  trace(navigateToQuoteListFromMyAccount.name, () => {
```

Comment:
Please, use your `log()` method instead of `trace()` one

Reply:
done